git-hooks/pre-commit

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+import sys
+from contextlib import contextmanager
```

Comment:
Some of these imports are no longer relevant -- can you remove them?

Reply:
Good catch, will do!